### PR TITLE
add error message for `pleg/generic.go:121`

### PR DESCRIPTION
**Pull request by batazor**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

Add an error message to log

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

copy from [container_manager.go](https://github.com/kubeedge/kubeedge/blob/f64b88469868b5b8a3c45d5a4d908e6749ed5097/edge/pkg/edged/containers/container_manager.go#L902)

### CODE REVIEW

File: edge/pkg/edged/pleg/generic.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (gl *GenericLifecycle) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kub
 
 	hostIP, err := gl.getHostIPByInterface()
 	if err != nil {
-		log.LOGGER.Errorf("Unable to get host IP")
+		log.LOGGER.Errorf("Cannot get host IP: %v", err)
```
Comment (fisherxu) [inline edge/pkg/edged/pleg/generic.go]: Thanks for add err msg! How about `Failed to get host IP`?

Comment (batazor) [reply edge/pkg/edged/pleg/generic.go]: Done @fisherxu

**Comment by fisherxu** [comment]
@batazor Could you please squash the commits, and let's get PR merged :)

**Comment by batazor** [comment]
@fisherxu

**Comment by batazor** [comment]
@rohitsardesai83

**Comment by sids-b** [comment]
@rohitsardesai83, you need to approve this PR to get it merged

**Pull request merged**